src/screens/ForgetPasswordScreen/ForgetPasswordScreen.tsx

```diff
@@ -3,12 +3,10 @@ import React, {useEffect, useState} from "react";
 import {useDispatch, useSelector} from "react-redux";
 import {forgetPassword} from "../../redux/actions/forgetPassword";
 import {useNavigation} from "@react-navigation/native";
-import {CodeVerify} from "../CodeVerifyScreen/CodeVerify";
 import {emailValidationSchema} from "../LoginScreen/validationScheme";
 import {Formik} from "formik";
 import {styles} from "./styles";
 import {Loader} from "../../components/Loader";
-import Animated, {} from "react-native-reanimated";
 import {displayErrorMessage} from "../../utils/displayMessage";
 import {RootState} from "../../redux/reducers/rootReducer";
 import {AnimatedText} from "./AnimatedText";
```

Comment:
This component uses `loading` local state in conjunction with redux logic. We update our local state based on redux action progress. That's not good as we're basically now have two sources of truth. We should have `loading` state in our redux (just like error that we already have here). See https://redux.js.org/tutorials/fundamentals/part-7-standard-patterns#async-request-status